pyside6_overlay/ui/panel_factory.py

```
@@ -0,0 +1,109 @@
+            return self.create_placeholder_panel(panel_id)
+            
+    def create_placeholder_panel(self, panel_id: str) -> QWidget:
+        """Create a placeholder panel when panel creation fails"""
+        from PySide6.QtWidgets import QLabel, QVBoxLayout
+        
+        placeholder = CardWidget()
+        placeholder.setObjectName(f"{panel_id}_placeholder")
+        
+        layout = QVBoxLayout(placeholder)
+        label = QLabel(f"Panel: {panel_id}")
+        label.setAlignment(Qt.AlignmentFlag.AlignCenter)
```

Ruff (0.11.9)

Undefined name `Qt`

(F821)